data-processing/entities/sentences/extractor.py

```diff
@@ -20,6 +24,10 @@ class SentenceExtractor(EntityExtractor):
     """
 
     def parse(self, tex_path: str, tex: str) -> Iterator[Sentence]:
+        for reserved_char in PSYDB_RESERVED_CHARACTERS:
+            if reserved_char in tex:
+                logging.warning('reserved character "%s" found in tex string, this might break the sentence extractor.', reserved_char)
```

Comment:
```suggestion
                logging.warning('Reserved character from pysbd "%s" found in tex string, this might break the sentence extractor.', reserved_char)
```